.github/workflows/linux.yml

```diff
@@ -24,7 +24,7 @@ on:
 jobs:
   linux:
     name: Check Linux
```

Comment:
this would deserve a more descriptive name too. I am not even sure what it tests. The build harness?

Reply:
if it doesn't test NB itself, this job might be able run on JDK 17, since the basic cluster should build ok by now.